build.gradle

```diff
@@ -58,29 +55,26 @@ version = '0.0.1-SNAPSHOT'
 
 description = ''
 
-bootRepackage {
-   mainClass = 'com.icthh.xm.ms.balance.BalanceApp'
+bootWar {
+    mainClassName = 'com.icthh.xm.ms.balance.BalanceApp'
 }
 
 war {
 
 }
 
 springBoot {
-    mainClass = 'com.icthh.xm.ms.balance.BalanceApp'
-    executable = true
-    buildInfo() {
-        additionalProperties = [
```

Comment:
As far as i remember that was a task to show this information on the monitoring page. Was it deleted deliberately?

Reply:
fixed